Week 2 - Arrays/readability/readability.c

```diff
@@ -0,0 +1,63 @@
+#include <cs50.h>
+#include <ctype.h>
+#include <math.h>
+#include <stdio.h>
+#include <string.h>
+
+int getLevel(string text);
+
+int main(void)
+{
+    string text = get_string("Text: ");
+
+    int level = getLevel(text);
```

Comment:
unfortunately snake case is the only option for cs50 and c 😄 but in Swift we use only camelCase ❤️
```suggestion
    int level = get_level(text);
```